README.md

```diff
@@ -44,10 +44,9 @@ class MyTest {
 }
 ```
 
-The `ExasolTestSetupFactory` in the example return a test setup depending on the `EXASOL_TEST_BACKEND` environment variable. Possible values:
+The `ExasolTestSetupFactory` in the example return a test setup for an AWS cluster if the specified config file (`cloudSetup/generated/testConfig.json`) exists. Otherwise, it returns a test setup based on a local running Exasol-Testconatiner.
```

Comment:
```suggestion
The `ExasolTestSetupFactory` in the example returns a test setup for an AWS cluster if the specified config file (`cloudSetup/generated/testConfig.json`) exists. Otherwise, it returns a test setup based on a local running Exasol-Testcontainer.
```